### PR TITLE
Created work section in portfolio project using grid

### PR DESCRIPTION
**Pull request by Pazzo97**
This Pull Request is about the work section that I have  added to portfolio project by using grids

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -116,6 +116,291 @@ <h5>Skills</h5>
...
+
+              <p>CSS <img class="rectangle" src="images/Rectangle70.png" alt="rectangle">
+
+                Html <img class="rectangle" src="images/Rectangle70.png" alt="rectangle">
+
+                Bootstrap <img class="rectangle" src="images/Rectangle70.png" alt="rectangle">
+
+                Ruby
+
+              </p>
```
Comment (smunozmo) [inline index.html]: - [ ] Good job here, your design looks great, however, it is important to use `ul` and `li` tags to list these technologies e.g:

```
<ul>
    <li>CSS</li>
    <li>HTML</li>
    <li>Ruby</li>
</ul>

```

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Pazzo97 , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

[OPTIONAL] - When coding with HTML it is a better practice to avoid empty blank lines, just use them to improve readability, here's an example, keep in mind this is just a friendly suggestion to make your code look even better:

Captura de pantalla 2022-03-18 a la(s) 8 16 35 p  m


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -116,6 +103,235 @@ <h5>Skills</h5>
...
+                <li><img class="rectangle" src="images/Rectangle70.png" alt="rectangle"></li>
+                <li>Bootstrap</li>
+                <li><img class="rectangle" src="images/Rectangle70.png" alt="rectangle"></li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+
+            <div class="story-button">
+              <a href="#" class="button button-1">See Project</a>
+            </div>
```
Comment (AmaduKamara) [inline index.html:166]: [ Optional Changes ] These changes won't stop your project from being approved but will help improve your codebase and set you up to become a better developer that follows HTML5 standards.

- [ ] Kindly use a `button` tag here or an `a` alone. I see you are using a div and styling it to look like a button but the idea here is for you to either use a `button` tag or an `a` tag. 
- Hint: You can instead write it this way: 
`<buttonclass="story-button">
              <a href="#" class="button button-1">See Project</a>
            </button>`
OR 
You use just `<a href="#" class="button button-1" class="button button-1">See Project</a>`

File: index.html [hunk 3]
```diff
@@ -116,6 +103,235 @@ <h5>Skills</h5>
...
```
Comment (AmaduKamara) [inline index.html:200]: [ Optional Changes ] These changes won't stop your project from being approved but will help improve your codebase and set you up to become a better developer that follows HTML5 standards.

- [ ] Kindly use a `button` tag here or an `a` alone. I see you are using a div and styling it to look like a button but the idea here is for you to either use a `button` tag or an `a` tag. 
- Hint: You can instead write it this way: 
`<buttonclass="story-button">
              <a href="#" class="button button-1">See Project</a>
            </button>`
OR 
You use just `<a href="#" class="button button-1" class="button button-1">See Project</a>`

File: index.html [hunk 4]
```diff
@@ -116,6 +103,235 @@ <h5>Skills</h5>
...
```
Comment (AmaduKamara) [inline index.html:236]: [ Optional Changes ] These changes won't stop your project from being approved but will help improve your codebase and set you up to become a better developer that follows HTML5 standards.

- [ ] Kindly use a `button` tag here or an `a` alone. I see you are using a div and styling it to look like a button but the idea here is for you to either use a `button` tag or an `a` tag. 
- Hint: You can instead write it this way: 
`<buttonclass="story-button">
              <a href="#" class="button button-1">See Project</a>
            </button>`
OR 
You use just `<a href="#" class="button button-1" class="button button-1">See Project</a>`

File: index.html [hunk 5]
```diff
@@ -116,6 +103,235 @@ <h5>Skills</h5>
...
```
Comment (AmaduKamara) [inline index.html:271]: [ Optional Changes ] These changes won't stop your project from being approved but will help improve your codebase and set you up to become a better developer that follows HTML5 standards.

- [ ] Kindly use a `button` tag here or an `a` alone. I see you are using a div and styling it to look like a button but the idea here is for you to either use a `button` tag or an `a` tag. 
- Hint: You can instead write it this way: 
`<buttonclass="story-button">
              <a href="#" class="button button-1">See Project</a>
            </button>`
OR 
You use just `<a href="#" class="button button-1" class="button button-1">See Project</a>`

File: index.html [hunk 6]
```diff
@@ -116,6 +103,235 @@ <h5>Skills</h5>
...
+            <input type="email" name="email" placeholder="jeremy@gmail.com"><br><br>
+          </div>
+
+          <div class="text-area">
+            <textarea placeholder="Enter text here"></textarea>
+          </div>
+
+          <div class="contact-btn">
+            <a href="#" class="button button-1">See Project</a>
+          </div>
```
Comment (AmaduKamara) [inline index.html:304]: [ Optional Changes ] These changes won't stop your project from being approved but will help improve your codebase and set you up to become a better developer that follows HTML5 standards.

- [ ] Kindly use a `button` tag here or an `a` alone. I see you are using a div and styling it to look like a button but the idea here is for you to either use a `button` tag or an `a` tag. 
- Hint: You can instead write it this way: 
`<buttonclass="story-button">
              <a href="#" class="button button-1">See Project</a>
            </button>`
OR 
You use just `<a href="#" class="button button-1" class="button button-1">See Project</a>`

**Review by AmaduKamara — APPROVED** [review]
# STATUS: APPROVED 👍 

Hi @Pazzo97, 

You are doing great. Keep rocking 👍
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**